db/models/cart.js

```diff
@@ -0,0 +1,82 @@
+const client = require("./client");
```

Comment:
- require client statement should have 2 dots to reference a file outside the immediate directory.

Reply:
```suggestion
const client = require("../client");
```